src/dymaptic.GeoBlazor.Core/Components/Layers/ImageryLayer.cs

```diff
@@ -0,0 +1,143 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Renderers;
+using dymaptic.GeoBlazor.Core.Exceptions;
+using dymaptic.GeoBlazor.Core.Serialization;
+using Microsoft.AspNetCore.Components;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Components.Layers;
+
+/// <summary>
+///     Represents a dynamic image service resource as a layer. An ImageryLayer retrieves and displays data from dynamic image services. An image service supports
+///     accessing the mosaicked image, its catalog, and the individual rasters in the catalog. An image service supports dynamic access and tiled access. 
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-ImageryLayer.html">
+///         ArcGIS
+///         JS API
+///     </a>
+/// </summary>
+public class ImageryLayer : Layer
+{
+    /// <inheritdoc />
+    [JsonPropertyName("type")]
+    public override string LayerType => "imagery";
+
+    /// <summary>
+    ///     Parameterless constructor for use as a razor component
+    /// </summary>
+    public ImageryLayer()
+    {
+    }
+
+    /// <summary>
+    ///     Constructor for use in code
+    /// </summary>
+    /// <param name="url"></param>  
+    /// <param name="portalItem"></param>
+    /// <param name="renderer"></param>
+    /// <param name="format"></param>
```

Comment:
Either fill out or delete the `<param>` lines please

Reply:
filled out the comments and removed the format parameter that was removed.